tests/python/frontend/pytorch/test_forward.py

```diff
@@ -578,9 +578,15 @@ class Squeeze2(Module):
         def forward(self, *args):
             return args[0].squeeze(1)
 
+    class Squeeze3(Module):
+        def forward(self, *args):
+            return args[0].squeeze((1, 3))
+
     input_data = torch.rand(input_shape).float()
     verify_model(Squeeze1().float().eval(), input_data=input_data)
     verify_model(Squeeze2().float().eval(), input_data=input_data)
+    if package_version.parse(torch.__version__) >= package_version.parse("2.0.0"):
```

Comment:
Why did you add this check on torch version?

Reply:
Because support for list of int for axis is only in Pytorch 2. In Pytorch 1.x, its only 1 int.

Reply:
Got it, thank you!